mtags/src/main/scala-2/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -274,11 +276,22 @@ class CompletionProvider(
       def isNotLocalForwardReference: Boolean =
         !head.sym.isLocalToBlock ||
           !head.sym.pos.isAfter(pos)
+      def isFileAmmoniteCompletion() = {
```

Comment:
Super nitpicky, but wrapping the expression in brackets isn't necessary here. I only bring it up because we don't do it for the method above. If you prefer it, then no worries.

Reply:
Fixed!